app/src/main/java/org/groundplatform/android/ui/common/BindingAdapters.kt

```diff
@@ -52,14 +52,11 @@ object BindingAdapters {
     if (view == null || uri == null) return
 
     try {
-      Picasso.get()
```

Comment:
Can we cleanup the gradle dependency too?

Reply:
https://github.com/google/ground-android/issues/3193

Reply:
Are there any other usages of Picasso left? Can we do it in the same PR?

Reply:
removed